lib/sycamore/sycamore/docset.py

```diff
@@ -967,6 +972,25 @@ def sort(self, descending: bool, field: str, default_val: Optional[Any] = None)
 
         return DocSet(self.context, Sort(self.plan, descending, field, default_val))
 
+    def groupby_count(self, field: str, unique_field: Optional[str] = None, **kwargs) -> "DocSet":
+        """
+        Performs a count aggregation on a DocSet.
+
+        Args:
+            field: Field to aggregate based on.
+            unique_field: Determines what makes a unique document.
+            **kwargs
+
+        Returns:
+            A DocSet with "properties.key" (unique values of document field)
+            and "properties.count" (frequency counts for unique values).
+        """
+        from sycamore.transforms import GroupByCount
```

Comment:
Just curious, why do we do these dynamic imports instead of importing them at the top of the file?

Reply:
Not fully sure... was just following the pattern